CONTRIBUTING.md

```diff
@@ -32,6 +32,7 @@ Stakeholders for `Convene` fall into one or more of the following categories:
 - [`Operators`], who steward the [`Neighborhood`: Infrastructure], [`Neighborhood`: Trust & Safety], and [`Client`: Support] responsibilities
 - [`Contributors`] and [`Maintainers`], who design and build [`Furniture`].
 
+Note: `Furniture` has been renamed to `Gizmo/Gizmos`.
 
```

Comment:
I had to add the note to a LOT of places. Maybe it's overkill. what to ya'll think?

Reply:
I think this is fine, especially since you already did it!